src/main/asciidoc/preface.adoc

```diff
@@ -1,153 +1,134 @@
+with an action or terms defined in text.
+a|From the *File* menu, choose *Open Project*.
 
 A *cache* is a copy that is stored locally.
 
-|`Monospace` |Monospace type indicates the names of files and
-directories, commands within a paragraph, URLs, code in examples, text
-that appears on the screen, or text that you enter. a|
-Edit your `.login` file.
+|`Monospace`  
+|Monospace type indicates the names of files and directories, commands
+within a paragraph, URLs, code in examples, text that appears on the
+screen, or text that you enter. 
+a|Edit your `.login` file.
 
 Use `ls -a` to list all files.
 
 `_machine_name_% you have mail.`
 
-|_Italic_ |Italic type indicates book titles, emphasis, or placeholder
-variables for which you supply particular values. a|
-Read Chapter 6 in the _User's Guide_.
+|_Italic_ 
+|Italic type indicates book titles, emphasis, or placeholder variables
+for which you supply particular values.
+a|Read Chapter 6 in the _User's Guide_.
 
 Do _not_ save the file.
 
 The command to remove a file is `rm _filename_`.
+|===
 
-|=======================================================================
-
-
-[[GFIRK]][[default-paths-and-file-names]]
-
-== Default Paths and File Names
+=== Default Paths and File Names
 
 The following table describes the default paths and file names that are
 used in this book.
 
-[width="99%",cols="20%,38%,38%"]
-|=======================================================================
+[width="99%",cols="20%a,38%,38%a"]
+|===
 |*Placeholder* |*Description* |*Default Value*
-|`_as-install_` |Represents the base installation directory for GlassFish
-Server. a|
-Installations on the Solaris operating system, Linux operating system,
+|`_as-install_` 
+|Represents the base installation directory for GlassFish
+Server. 
+| Installations on the Solaris operating system, Linux operating system,
 and Mac operating system:
 
-`_user's-home-directory_/glassfish5/glassfish`
+`_user's-home-directory_/glassfish6/glassfish`
 
 Windows, all installations:
 
-`_SystemDrive_:\glassfish5\glassfish`
+`_SystemDrive_:\glassfish6\glassfish`
 
-|`_as-install-parent_` |Represents the parent of the base installation
-directory for GlassFish Server. a|
-Installations on the Solaris operating system, Linux operating system,
+|`_as-install-parent_` 
+|Represents the parent of the base installation
+directory for GlassFish Server. 
+|Installations on the Solaris operating system, Linux operating system,
 and Mac operating system:
 
-`_user's-home-directory_/glassfish5`
+`_user's-home-directory_/glassfish6`
 
 Windows, all installations:
 
-`_SystemDrive_:\glassfish5`
+`_SystemDrive_:\glassfish6`
 
-|`_tut-install_` |Represents the base installation directory for the Jakarta EE
-Tutorial after you download it. |`_as-install-parent_/docs/jakartaee-tutorial`
+|`_tut-install_` 
+|Represents the base installation directory for the Jakarta EE
+Tutorial after you download it. 
+|`_as-install-parent_/docs/jakartaee-tutorial`
```

Comment:
The tutorial is not bundled in GlassFish anymore so this reference should be removed.

Reply:
Could this be changed to "Represents the base installation directory for the Jakarta EE
Tutorial **after you download and extract it to the default location**".

Reply:
Or just "Represents the base installation directory for the Jakarta EE Tutorial" and third column as blank

Reply:
I have updated this to user's home directory.